src/domain/application/taydennysAndMuutosilmoitusCommon/MuutosLabelContext.tsx

```diff
@@ -0,0 +1,17 @@
+import { createContext, useContext } from 'react';
+
+const MuutosLabelContext = createContext<string | undefined>(undefined);
+
+export function MuutosLabelProvider({
+  children,
+  value,
+}: {
+  children: React.ReactNode;
```

Comment:
My IDE marks this as warning and advices to add `import React from 'react'`.

Reply:
Added the import. For some reason my IDE did not warn about it.

Reply:
Also, SonarCloud says that `Mark the props of the component as read-only.`
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1111&id=City-of-Helsinki_haitaton-ui&open=AZVlUIuUxq_vcSl6uWOX